qa/Rakefile

```diff
@@ -18,36 +18,47 @@ namespace :test do
     end
   end
 
-  desc "Bootstrap all the VM's used for this tests"
-  task "setup" do
-    puts "bootstraping all VM's defined in acceptance/Vagrantfile"
-    cd "acceptance" do
-      LogStash::VagrantHelpers.bootstrap
-    end
-  end
-
   namespace :acceptance do
 
     desc "Run all acceptance"
     task :all do
-      exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/**/*_spec.rb"]]))
+      all_boxes = LogStash::VagrantHelpers::DEFAULT_CENTOS_BOXES + LogStash::VagrantHelpers::DEFAULT_DEBIAN_BOXES 
+      for box in all_boxes do
+        LogStash::VagrantHelpers.bootstrap(box)
```

Comment:
I'm on the fence here, my thoughts at the beginning where to design this as decoupled as possible so:

* Running test does nothing to do with bootstrapping the VM's.
* We get the machines bootstrapped/destroyed (__with rake tasks__), but all at once at the beginning of the process.

I guess the main idea here is to make this process as fast as possible, have you tested which approach is:

* Easier to operate.
* Faster to run the test, not just one but might be more than one. The common workflow I see here is to run all test for each platform (aka _debian_, _centos_, ....)
* Let's you investigate what went wrong if a test fails.

what do you think?